Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -1862,5 +2021,419 @@ public void DeclineLobbyInvite()
+        }
+
+        /// <summary>
+        /// Whenever a user attempts to create a connection, this method handles their connection request.
+        /// By default, accept all incoming connections.
+        /// </summary>
+        /// <param name="data">Data, containing the product user id of the connecting request.</param>
+        private void OnIncomingSessionsConnectionRequest(ref OnIncomingConnectionRequestInfo data)
+        {
+            if (!(bool)data.SocketId?.SocketName.Equals(P2PSessionStatusSocketName))
+            {
+                Debug.LogError($"EOSSessionsManager (OnIncomingSessionsConnectionRequest): This function should not be handling this message, its socket is not '{P2PSessionStatusSocketName}'. Socket name is '{(data.SocketId?.SocketName)}'.");
+                return;
+            }
+
+            SocketId socketId = new SocketId()
+            {
+                SocketName = P2PSessionStatusSocketName
+            };
+
+            AcceptConnectionOptions options = new AcceptConnectionOptions()
+            {
+                LocalUserId = EOSManager.Instance.GetProductUserId(),
+                RemoteUserId = data.RemoteUserId,
+                SocketId = socketId
+            };
+
+            Result result = EOSManager.Instance.GetEOSPlatformInterface().GetP2PInterface().AcceptConnection(ref options);
+
+            if (result != Result.Success)
+            {
+                Debug.LogErrorFormat("EOSSessionsManager (OnIncomingSessionsConnectionRequest): Eror while accepting connection, code: {0}", result);
+            }
+            else
+            {
+                Log($"EOSSessionsManager (OnIncomingSessionsConnectionRequest): Successfully accepted connection from {options.RemoteUserId} on socket {P2PSessionStatusSocketName}");
+            }
+        }
+
+        /// <summary>
+        /// Upon a user that is connected disconnecting, this method closes out the connection.
+        /// </summary>
+        /// <param name="data">Data, containing the product user id of the user to disconnect from.</param>
+        private void OnIncomingSessionsDisconnect(ref OnRemoteConnectionClosedInfo data)
+        {
+            SocketId socketId = new SocketId()
+            {
+                SocketName = P2PSessionStatusSocketName
+            };
+
+            CloseConnectionOptions closeOptions = new CloseConnectionOptions()
+            {
+                LocalUserId = EOSManager.Instance.GetProductUserId(),
+                RemoteUserId = data.RemoteUserId,
+                SocketId = socketId
+            };
+
+            Result result = EOSManager.Instance.GetEOSPlatformInterface().GetP2PInterface().CloseConnection(ref closeOptions);
+
+            if (result != Result.Success)
+            {
+                Debug.LogErrorFormat("EOSSessionsManager (OnIncomingSessionsDisconnect): Eror while closing connection, code: {0}", result);
```

Comment:
This was mostly following the conventions that were established in this file already. I'm very comfortable changing things over to the $"" string interpolation. Shall I pull the trigger on changing those here?

Reply:
I would pull the trigger on making those changes for new implementations. I understand how this is somewhat subjective, because other guidance regarding refactors dictates that you follow the format and styling that already exists within the file. 

I would clarify that strategy to say that you should always match existing style and formatting _if_ that style and formatting is consistent. If it's _already_ not consistent, then introduce changes that only include style and format decisions that are consistent internal to the changes being made (in other words, don't continue to use inconsistent styling, try to pick the best one shown in the file and follow that one everywhere your changes touch).